### ISSUE TITLE
SSD windows

### ISSUE THREAD

**Issue by MyVanitar** [dtmoodie/caffe#13]
Hello,

As you know there is no repository to train and run the SSD on Caffe windows. Does your repo helps about this?

**Comment by dtmoodie** [comment]
Hello,

The merge branch has successfully trained SSD on windows, and successfully been used for inference.

**Comment by MyVanitar** [comment]
Hello,

Thank you for the reply.

Would you please update the repo? Because Caffe-SSD has improved some parts of its original caffe.

**Comment by dtmoodie** [comment]
I've been considering an update for a while.
I have another branch (sanghoon-dev_pvanet) which has pvanet and SSDv2 merged.  However this also has merges from nvidia's fork (github.com/nvidia/caffe) which improves training performance significantly, but breaks inference on Windows. (https://github.com/NVIDIA/caffe/issues/284)
At some point I will update a non-nvidia branch, but I don't have time right now I'm currently working with darknet and mxnet.

**Comment by MyVanitar** [comment]
Are you working on the YOLO v2? 

There are two unknown points in YOLO, one is anchors and another one is transfer learning. Many people think training with an initial weights is like fine-tuning but it is not. The result accuracy proves it also that it is training fro scratch.

I selected the SSD because it showed higher accuracy and speed but PVANET+ is better than all of these two, but the provided repo of PVANET+ seems to not be the one used to publish the paper, as authors mentioned.

**Comment by MyVanitar** [comment]
Also please fill out the `readme` section in the Github. I checked the `dev_pvanet` also and but it has very short `readme`. Many people including me don't know how to use your repo, although it might be a very nice and excellent work.

**Comment by dtmoodie** [comment]
Yes,
I've found that YOLOv2 is better at picking up small objects at the framerate that I need since I can run it at 1088 resolution vs SSD's 512 at the same speed.

**Comment by MyVanitar** [comment]
Do you know how to fine-tune the YOLO or calculate the anchors?

**Comment by dtmoodie** [comment]
I roughly followed the instructions here:
https://github.com/AlexeyAB/darknet
Anchor generation here:
https://github.com/Jumabek/darknet_scripts/blob/master/gen_anchors.py
